plugins/MultiSites/vue/src/AllWebsitesDashboard/AllWebsitesDashboard.vue

```diff
@@ -16,6 +16,31 @@
     </h1>
   </div>
 
+  <div v-if="errorLoading">
+      <div class="notification system notification-error">
+        {{ translate('MultiSites_AllWebsitesDashboardErrorMessage', '', '') }}
```

Comment:
We don't need to pass the placeholder values since the translatable string doesn't have any placeholders (unlike the original general error message).
```suggestion
        {{ translate('MultiSites_AllWebsitesDashboardErrorMessage') }}
```

Reply:
fixed.